### PR TITLE
I hate it, but it works

### PR DESCRIPTION
**Pull request by TAImatem**
it at least fixes Raicuparta's SpawnPoints

### CODE REVIEW

**Comment by TAImatem** [comment]
we either do this or recompile mods

File: OWML.Common/Menus/IModPauseMenu.cs [hunk 1]
```diff
@@ -2,7 +2,10 @@
 {
     public interface IModPauseMenu : IModPopupMenu, IModOWMenu
     {
-        IModTitleButton ResumeButton { get; }
+        IModTabbedMenu OptionsMenu { get; }
```
Comment (amazingalek) [inline OWML.Common/Menus/IModPauseMenu.cs:5]: Was this needed?

Comment (TAImatem) [reply OWML.Common/Menus/IModPauseMenu.cs:5]: yep, same for main menu. Not sure why it doesn't get resolved to IModOWMenu

Comment (TAImatem) [reply OWML.Common/Menus/IModPauseMenu.cs:5]: (yes, we now have a ton of compile warnings :shrug:

File: OWML.Common/Menus/IModMenu.cs [hunk 2]
```diff
@@ -10,18 +10,20 @@ public interface IModMenu
...
         [Obsolete("Use GetTitleButton instead")]
-        IModTitleButton GetButton(string title);
-        IModTitleButton GetTitleButton(string title);
+        IModButton GetButton(string title);
+        IModButton GetTitleButton(string title);
         IModPromptButton GetPromptButton(string title);
 
         IModButton AddButton(IModButton button);
         IModButton AddButton(IModButton button, int index);
+        IModButtonBase AddButton(IModButtonBase button);
```
[the remark this answers is not in the document]
Comment (TAImatem) [reply OWML.Common/Menus/IModMenu.cs:25]: you're wrong :P
I had to do it while testing for SpawnPoint mod

**Pull request merged**